config/plugins/visualizations/jqplot/jqplot_bar/config/jqplot_bar.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE visualization SYSTEM "../../visualization.dtd">
-<visualization name="Bar diagram (jqPlot)">
+<visualization name="Bar diagram (jqPlot)" visible="false">
```

Comment:
I think? for tools this is hidden instead of visible, probably good to be consistent here.